app/scripts/modules/openstack/common/validateType.directive.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.openstack.validateType.directive', [
+])
+.directive('validateType', [
+  function() {
+
+    var link = function($scope, $element, $attrs, ctrl) {
+
+      var validate = function(viewValue) {
+        var comparisonModel = $attrs.validateType;
+
+        if(parseInt(viewValue, 10) === -1 && parseInt(comparisonModel, 10) > -1)
```

Comment:
What does it mean if `viewValue` is `-1` and `comparisonModel` is greater than `-1`?